pkg/gui/keybindings/keybindings.go

```diff
@@ -11,6 +11,10 @@ import (
 	"github.com/jesseduffield/lazygit/pkg/gui/types"
 )
 
+const (
+	nop = "<nop>"
```

Comment:
I'd add a comment saying that 'this is a special key which allows the user to disable keybindings'